makefiles/common.mk

```diff
@@ -142,12 +142,12 @@ bench/cleanup:
 ## executes a dry run of the release process
 .PHONY: release/dry-run
 release/dry-run:
-	go run github.com/goreleaser/goreleaser@$(GO_RELEASER_VERSION) release --snapshot --rm-dist
+	goreleaser release --snapshot --rm-dist 
 
 ## generates a terramate release
 .PHONY: release
 release:
-	go run github.com/goreleaser/goreleaser@$(GO_RELEASER_VERSION) release --rm-dist
```

Comment:
this variable is now unused in the makefile.
I'm approving this now but it would be good if you can improve the Makefile in the future by checking if the installed version of goreleaser matches the version configured in the Makefile.